src/uct/base/uct_md.c

```diff
@@ -319,9 +319,36 @@ ucs_status_t uct_config_modify(void *config, const char *name, const char *value
     return ucs_config_parser_set_value(bundle->data, bundle->table, name, value);
 }
 
+static ucs_status_t
+uct_md_mkey_pack_params_check(const uct_md_mkey_pack_params_t *params)
+{
+#if ENABLE_PARAMS_CHECK
+    return ((uct_md_mkey_pack_param_memh(params) != NULL) &&
```

Comment:
```c
return (ENABLE_PARAMS_CHECK && 
        (uct_md_mkey_pack_param_memh(params) != NULL) &&
        (uct_md_mkey_pack_param_rkey_buf(params) != NULL)) ?
       UCS_OK : UCS_ERR_INVALID_PARAM;
```

Reply:
this returns `UCS_ERR_INVALID_PARAM`  if `ENABLE_PARAMS_CHECK  == 0`
correct 1-line-condition IMO looks overcomplicated

Reply:
ok, but can we use ENABLE_PARAMS_CHECK as a runtime check instead of compile time check? with 'if' condition?
IMO code is simpler to read without ifdef